pyensembl/gtf_parsing.py

```diff
@@ -121,20 +122,30 @@ def _extend_with_attributes(df):
     extra_columns = {}
     column_order = []
     for i, attr_string in enumerate(df.attribute):
+        # TODO: tokenize attribute strings properly
+        # for now, catch mistaken semicolons by replacing "xyz;" with "xyz"
+        # Required to do this since the GTF for Ensembl 78 has
+        # gene_name = "PRAMEF6;"
+        # transcript_name = "PRAMEF6;-201"
+        attr_string = attr_string.replace(';\"', '\"').replace(";-", "-")
+
         pairs = (
             kv.strip().split(" ", 2)
             for kv in attr_string.split(";")
-            if len(kv) > 0
+            # simplest entry must be at least three characters:
+            # (key name, space, value)
```

Comment:
Wait, so if " " splits key and value, what does ";" do? And if it splits key value elements from other key value elements, why is "gene_name = "PRAMEF6;" above wrong? I find this little bit confusing